server/routes.go

```diff
@@ -36,7 +36,7 @@ func newEventFromRequest(r *http.Request) (*types.Event, error) {
 	if !form.Has("created_at") {
 		form.Set("created_at", time.Now().UTC().Format("2006-01-02 15:04:05"))
 	}
-	return types.NewEvent(form, r.Header, r.UserAgent()), nil
+	return types.NewEvent(form, r.Header, r.UserAgent(), r.RemoteAddr), nil
```

Comment:
Do we know how if `r.RemoteAddr` will contain the real IP if our collector is behind a load balancer?
I know that load balancers expose a header `x-forwarded-for` that helps us keep track of the Original Remote IP address.

Reply:
I applied your review comments. Please, check. If needed we can have a talk.